packages/app/src/device-info/services/device-info.service.ts

```diff
@@ -30,4 +33,37 @@ export class DeviceInfoService {
       return Result.failed(new AppError("", ""))
     }
   }
+
+  public async getDeviceFreeSpace(): Promise<ResultObject<number, string>> {
+    try {
+      const response = await this.deviceManager.device.request<
+        DeviceInfoRaw | NotSupportedDeviceInfo
+      >({
+        endpoint: Endpoint.DeviceInfo,
+        method: Method.Get,
+      })
+
+      if (!response.ok) {
+        return response
+      }
+
+      const freeSpaceInNoTSupportedDevice = +(
```

Comment:
🟡 
Please use Number() 🙏  The casting (conversion of one data type to another) is making explicitly for this project.